spec/features/user_can_loginAndlogout_spec.rb

```diff
@@ -0,0 +1,50 @@
+feature "User can login" do
+    let(:user) { create(:user) }
```

Comment:
incosistent indentation in this file, make sure that you use a code formatter like rubocop or similar

Reply:
@faraznaeem

> incosistent indentation in this file, make sure that you use a code formatter like rubocop or similar

Can we have a session about Rubocop ?